subprojects/gradle/module-dependencies-shared/src/main/kotlin/com/avito/module/dependencies/ModuleConfigurationDependencies.kt

```diff
@@ -4,20 +4,21 @@ import com.avito.module.configurations.ConfigurationType
 import org.gradle.api.Project
 import org.gradle.api.artifacts.ProjectDependency
 
-public fun Project.dependenciesOnProjects(
-    types: Set<Class<out ConfigurationType>>
-): Set<ProjectDependency> {
+public fun Project.directDependenciesOnProjects(
+    types: Set<ConfigurationType>
+): Map<ConfigurationType, Set<Project>> {
```

Comment:
We lost a mapping before. 
This is a client's responsibility to decide what to do with this information.